docs/prepping-local-system-runbook-linux.md

```diff
@@ -546,69 +546,68 @@ It is a best practice to retag each of your Pega Docker images by including your
 
    `$ docker push <Registry host name:Port>/my-pega:8.4.0`
```

Comment:
can we update all of these tags to '24.1 now?
docker push <Registry host name:Port>/my-pega:24.1.0`